src/core/services/google-maps-types.ts

```diff
@@ -404,17 +415,26 @@ export interface Geometry {
   type: string;
 }
 
-export class ControlPosition{
-  readonly BOTTOM_CENTER: string = 'BOTTOM_CENTER';
-  readonly BOTTOM_LEFT: string = 'BOTTOM_LEFT';
-  readonly BOTTOM_RIGHT: string = 'BOTTOM_RIGHT';
-  readonly LEFT_BOTTOM: string = 'LEFT_BOTTOM';
-  readonly LEFT_CENTER: string = 'LEFT_CENTER';
-  readonly LEFT_TOP: string = 'LEFT_TOP';
-  readonly RIGHT_BOTTOM: string = 'RIGHT_BOTTOM';
-  readonly RIGHT_CENTER: string = 'RIGHT_CENTER';
-  readonly RIGHT_TOP: string = 'RIGHT_TOP';
-  readonly TOP_CENTER: string = 'TOP_CENTER';
-  readonly TOP_LEFT: string = 'TOP_LEFT';
-  readonly TOP_RIGHT: string = 'TOP_RIGHT';
+/**
+ * Identifiers used to specify the placement of controls on the map. Controls are
+ * positioned relative to other controls in the same layout position. Controls that
+ * are added first are positioned closer to the edge of the map.
+ */
+export enum ControlPosition {
+  BOTTOM_CENTER,
```

Comment:
Yeah this is on my todo list. The main idea was, that this project has control over it and defines the types that it really uses (mainly for internal usage) and only the attributes that it uses. Only exported interface should be complete (because the end-user uses it).

I completely agree that we should use the types package long-term. But that should be another PR.